pkg/proxy/conntrack/cleanup_test.go

```diff
@@ -55,13 +56,38 @@ const (
 	testNonServingEndpointIP = "10.240.1.5"
 	testDeletedEndpointIP    = "10.240.2.6"
 
+	// testOldEndpointPort is used to cover cases when endpoint changes port,
+	// but IP remains same.
+	testOldEndpointPort = 8080
+	testEndpointPort    = 9090
```

Comment:
Nit (as with all my nits, do with it what you will).
```suggestion
	testPreviousEndpointPort = 8080
	testCurrentEndpointPort  = 9090
```

This may be more explicit, but up to you. The current names seem pretty clear.